grafana/risingwave-user-dashboard.dashboard.py

```diff
@@ -744,6 +744,26 @@ def section_streaming(outer_panels):
                         ),
                     ],
                 ),
+                panels.timeseries_latency(
+                    "Latency of Materialize Views & Sinks",
```

Comment:
This seems to be easier to interpret than the epoch one. Should we add it to dev dashboard also? (Actually I've never checked user dashboard before... 🤪)